level-ttl.js

```diff
@@ -92,25 +75,29 @@ function startTtl (db, checkFrequency) {
         }
       })
   }, checkFrequency)
-  if (db._ttl.intervalId.unref)
+
+  if (db._ttl.intervalId.unref) {
     db._ttl.intervalId.unref()
+  }
 }
 
 function stopTtl (db, callback) {
   // can't close a db while an interator is in progress
   // so if one is, defer
-  if (db._ttl._checkInProgress)
-    return db._ttl._stopAfterCheck = callback
+  if (db._ttl._checkInProgress) {
+    db._ttl._stopAfterCheck = callback
+    return db._ttl._stopAfterCheck
```

Comment:
Can you add a TODO comment here, to check whether we need the return value? I'm guessing we don't